packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -63,8 +63,6 @@ export class SlackBot {
         this.analytics = analytics;
         this.slackAuthenticationModel = slackAuthenticationModel;
         this.unfurlService = unfurlService;
-
-        void this.start();
```

Comment:
Bad practice to run an async function in the constructor. We now call the `start` in App.ts.
This also allows this class methods to be overridden in classes that extend this one.